test/plugin/scenarios/mongodb-4.x-scenario/bin/startup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -jar ${agent_opts} ${home}/../libs/mongodb-4.x-scenario.jar &
```

Comment:
Please activate the parameter collection for testing. And the expected data should be updated too.

Reply:
> Please activate the parameter collection for testing. And the expected data should be updated too.

I don't quite understand, the integration test has been tested. Can you explain in more detail?

Reply:
The parameter in the MongoPluginConfig. Should be tested in there too.